src/OSPSuite.Core/CoreRegister.cs

```diff
@@ -57,6 +59,11 @@ public override void RegisterInContainer(IContainer container)
             scan.ExcludeType<ParameterIdentificationRunner>();
             scan.ExcludeType<SensitivityAnalysisRunner>();
 
+            scan.ExcludeType(typeof(ExtendedProperty<>));
```

Comment:
@Yuri05 Open type need to be registered separately. This works also with castle and this is in fact better practice